addons/website_slides/views/slide_channel_views.xml

```diff
@@ -107,40 +109,27 @@
+                                               domain="[('channel_id', '=', active_id), ('is_category', '=', False)]"/>
                                     </group>
                                 </group>
                                 <div attrs="{'invisible': [('enroll', '!=', 'invite')]}">
                                     <label for="enroll_msg"/>
                                     <field name="enroll_msg" colspan="4" nolabel="1"/>
                                 </div>
                             </page>
-                            <page string="Karma Rules" name="karma_rules">
-                                <group>
-                                    <field name="karma_gen_channel_rank" string="Rank Course"/>
-                                    <field name="karma_gen_channel_finish" string="Finish Course"/>
-                                    <field name="karma_review" attrs="{'invisible': [('allow_comment', '!=', True)]}"/>
-                                    <field name="karma_slide_comment" attrs="{'invisible': [('allow_comment', '!=', True)]}"/>
-                                    <field name="karma_slide_vote" attrs="{'invisible': [('allow_comment', '!=', True)]}"/>
-                                </group>
-                            </page>
-                            <page name="statistics" string="Statistics">
+                            <page string="Karma" name="karma_rules">
                                 <group>
-                                    <group name="statistics_nbr" class="col-4">
-                                        <field name="nbr_presentation"/>
-                                        <field name="nbr_document"/>
-                                        <field name="nbr_video"/>
-                                        <field name="nbr_infographic"/>
-                                        <field name="nbr_quiz" string="Quizzes"/>
-                                        <field name="nbr_webpage"/>
+                                    <group string="Rewards">
+                                        <field name="karma_gen_channel_rank" string="Review a Course"/>
+                                        <field name="karma_gen_channel_finish" string="Finish Course"/>
                                     </group>
-                                    <group name="view_stats">
-                                        <field name="total_views" string="Visits"/>
-                                        <field string="Rating" name="rating_avg" widget="field_float_rating"/>
-                                        <label for="total_time" string="Watch Time"/>
-                                        <div>
-                                            <field name="total_time" widget="float_time" />
-                                            <span class="ml-2">Hours</span>
-                                        </div>
```

Comment:
Are you sure removing statistics tab is in the spec ?

Reply:
Yes, as they are mostly duplicated else replaced by Stats buttons.